classes/order/OrderInvoice.php

```diff
@@ -129,6 +129,8 @@ public function add($autodate = true, $null_values = false)
 
     public function getProductsDetail()
     {
+        $id_lang = Context::getContext()->language->id;
```

Comment:
This line don't need to be merge if it's the only part taken of this commit: https://github.com/jocel1/PrestaShop/commit/eb237ec154febb5c03cd6243876f93e3896e6339

$id_lang  will be never used there.